shared/login/signup/request-invite.render.native.js

```diff
@@ -1,16 +1,62 @@
 /* @flow */
-/* eslint-disable react/prop-types */
 
 import React, {Component} from 'react'
-import {View} from 'react-native'
+import {globalStyles} from '../../styles/style-guide'
+import {Text, Icon, Input, Button, Box} from '../../common-adapters'
+import Container from '../forms/container'
 
-import type {Props} from './username-email-form.render'
+import type {Props} from './request-invite.render'
 
 export default class Render extends Component {
   props: Props;
 
   render () {
-    return <View />
+    return (
+      <Container onBack={this.props.onBack} style={stylesContainer}>
+        <Text style={stylesHeader} type='Header'>Request an invite code</Text>
+        <Icon style={stylesIcon} type='invite-code-m'/>
+        <Input
+          style={stylesInput}
+          hintText='Your email address'
+          value={this.props.email}
+          errorText={this.props.emailErrorText}
+          onChangeText={email => this.props.emailChange(email)}
+          autoFocus />
+        <Input
+          style={stylesInput}
+          hintText='Your name'
+          value={this.props.name}
+          errorText={this.props.nameErrorText}
+          onChangeText={name => this.props.nameChange(name)} />
+        <Button
+          fullWidth
+          style={stylesButton}
+          waiting={this.props.waiting}
+          type='Primary'
+          label='Request'
+          onClick={this.props.onSubmit}
+          disabled={!this.props.email} />
+        <Box style={{flex: 1}} />
+      </Container>
+    )
   }
 }
 
+const stylesButton = {
+  marginTop: 50
+}
```

Comment:
yeah we should discuss. i wish we could force eslint.. maybe we can. also we might have to use StyleSheet.create() on mobile, unclear if we're paying a serious penalty or not , not using it